src/api/boardsApi.js

```diff
@@ -0,0 +1,14 @@
+const fetch = require('node-fetch');
+
+function handleResponse(response) {
```

Comment:
This is duplicated to the `cardsApi.js`, so maybe try to eliminate that by extracting it to a generic api file, your call :blush:

Reply:
done, extracted one helpers.js file

Reply:
revert, breaking build for some reason